pkg/apis/core/v1beta1/defaults.go

```diff
@@ -181,6 +181,10 @@ func SetDefaults_Shoot(obj *Shoot) {
 		p := ShootPurposeEvaluation
 		obj.Spec.Purpose = &p
 	}
+
+	if obj.Spec.Kubernetes.Kubelet.FailSwapOn == nil {
```

Comment:
A small unit test in the `defaults_test.go` so that we don't accidentally break the API in the future would be great.

Reply:
I dont see a test for the SetDefaults_Shoot func... am I missing something?

Reply:
Hm, right, I've only added it as part of https://github.com/gardener/gardener/pull/2384/files#diff-86a47c7826e138a4538e48eb597c8846R267, so maybe you can follow the same approach and just add a test for your `failSwapOn` field for now.

Reply:
added